config/_default/config.toml

```diff
@@ -68,13 +68,15 @@ preserveTaxonomyNames = true
 version_menu = "Versions"
 
 [[params.versions]]
-fullversion = "master"
-version = "master"
-githubbranch = "master"
+fullversion = "v1.5"
```

Comment:
nit: Since this points to the latest version or the latest release It may be confusing to state the version number but not show the version number in the URL like we do for the v1.4.
Maybe we can consider using the tag `latest(v1.5)` . 
Overall LGTM.

Reply:
It is the same behavior as kubernetes does, see kubernetes.io.

Reply:
I understand it is the same as kubernetes.io. 
Just putting across my thoughts on the release version numbering for the Kubeedge website.